lesson08/assignment/video_cli/l8/src/top3.vue

```diff
@@ -1,36 +1,5 @@
 <template>
     <div>
-      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
The nav doesn't need to be repeated on each page since it is defined in the navbar component, that is called on every page since it is in App.vue.